seqr/views/utils/terra_api_utils.py

```diff
@@ -67,15 +77,23 @@ def is_anvil_authenticated(user):
 def _get_call_args(path, headers=None, root_url=None):
     if headers is None:
         headers = {"User-Agent": SEQR_USER_AGENT}
-    headers.update({"accept": "application/json"})
+    headers.update({"accept": "application/json", "Content-Type": "application/json"})
     if root_url is None:
         root_url = TERRA_API_ROOT_URL
     url = urljoin(root_url, path)
     return url, headers
 
 
+def _safe_get_social(user):
+    if not google_auth_enabled():
+        return None
```

Comment:
does this case get tested anywhere?

Reply:
Yes, when a non-anvil user logging out, it will be called from `remover_token()`.